data_adapter_oemof/calculations.py

```diff
@@ -34,3 +39,145 @@ def get_name(*args, counter=None):
 @calculation
 def get_capacity_cost(overnight_cost, fixed_cost, lifetime, wacc):
     return annuity(overnight_cost, lifetime, wacc) + fixed_cost
+
+
+def decommission(adapter_dict: dict) -> dict:
+    """
+
+    Takes adapter dictionary from adapters.py with mapped values.
+
+    I:
+    Takes largest found capacity and sets this capacity for all years
+    Each yearly changing capacity value is divided by max capacity and
+    quotient from `max capacity`/`yearly capacity` is set as max value.
+
+    II:
+    If Max value is already set by another parameter function will issue info
+    Recalculating max value to max_new = (max_old * capacity)/`the largest capacity`
+    Overwriting max value in `output_parameters`
+    Then is setting capacity to the largest found capacity
+
+    Supposed to be called when getting default parameters
+    Non investment objects must be decommissioned in multi period to take end of lifetime
+    for said objet into account.
+
+    Returns
+    adapter_dictionary with max values in output parameters and a single capacity
+    -------
+
+    """
+
+    def multiply_two_lists(l1, l2):
+        """
+        Multiplies two lists
+
+        Lists must be same length
+
+        Parameters
+        ----------
+        l1
+        l2
+
+        Returns divided list
+        -------
+
+        """
+        return [i * j for i, j in zip(l1, l2)]
+
+    capacity_column = "capacity"
+    max_column = "max"
+
+    # check if capacity column is there and if it has to be decommissioned
+    if capacity_column not in adapter_dict.keys():
+        logging.info("Capacity missing for decommissioning")
+        return adapter_dict
+
+    if not isinstance(adapter_dict[capacity_column], list):
+        logging.info("No capacity fading out that can be decommissioned.")
+        return adapter_dict
+
+    # I:
+    if max_column not in adapter_dict["output_parameters"].keys():
+        adapter_dict["output_parameters"][max_column] = adapter_dict[
+            capacity_column
+        ] / np.max(adapter_dict[capacity_column])
+    # II:
+    else:
+        logging.info("Decommissioning and max value can not be set in parallel")
```

Comment:
This sounds like an error?
If it is an actual error, you could/should raise an exception.
Or do you want to go on as you are expecting errors but want to process data anyway? In this case you should at least increase logging level to error.

Reply:
The calculation should enable setting decommission and max value in parallel, this logging info I forgot to delete as I am now calculating 
$max_{new} = \frac{(max_{column} \cdot capacity_{column})}{capacity_{max}}$